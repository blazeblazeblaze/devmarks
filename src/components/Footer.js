import React from "react";

const Footer = () => (
  <footer className="footer">
    <div className="container">
      <div className="content has-text-centered">
        <a
          target="_blank"
          href="https://github.com/blazejhadzik/devmarks"
          className="button is-outlined"
        >
          <span className="icon has-text-danger">
            <i className="fas fa-heart" />
          </span>
          <span>devmarks</span>
        </a>
      </div>
    </div>
  </footer>
);

export default Footer;
