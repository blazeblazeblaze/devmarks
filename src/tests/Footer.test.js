import React from "react";
import Footer from "../components/Footer";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";

configure({ adapter: new Adapter() });

describe("<Footer />", () => {
  it("matches snapshot", () => {
    const tree = shallow(<Footer />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
