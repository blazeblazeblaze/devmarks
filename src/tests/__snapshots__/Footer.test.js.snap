// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> matches snapshot 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <div
      className="content has-text-centered"
    >
      <a
        className="button is-outlined"
        href="https://github.com/blazejhadzik/devmarks"
        rel="noopener noreferrer"
        target="_blank"
      >
        <span
          className="icon has-text-danger"
        >
          <i
            className="fas fa-heart"
          />
        </span>
        <span>
          devmarks
        </span>
      </a>
    </div>
  </div>
</footer>
`;
